tests/robot-tests/tests/admin_and_public/bau/release_and_publication_role_permissions.robot

```diff
@@ -0,0 +1,352 @@
+    [Tags]  HappyPath
+    user changes to analyst1
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}  120
+    user navigates to release summary from admin dashboard  ${PUBLICATION_NAME}
+    ...  ${RELEASE_TYPE} (not Live)
+
+Navigate to 'Footnotes' section
+    [Tags]  HappyPath
+    user clicks link  Footnotes
+    user waits until h2 is visible  Footnotes
+
+Add a Footnote as a release owner
+    [Tags]  HappyPath
+    user waits until page contains link   Create footnote
+    user clicks link  Create footnote
+    user waits until h2 is visible  Create footnote
+    user clicks footnote radio   ${SUBJECT_NAME}   Applies to all data
+    user clicks element  id:footnoteForm-content
+    user enters text into element  id:footnoteForm-content  A footnote as analyst1 User1 with the release owner role
+    user clicks button  Save footnote
+    user waits until h2 is visible  Footnotes
+
+Assert release owner can create a release note
+    [Tags]  HappyPath
+    user clicks link  Content
+    user clicks button   Add note
+    user enters text into element  id:createReleaseNoteForm-reason  Test release note one
+    user clicks button   Save note
+    ${date}=  get current datetime   %-d %B %Y
+    user waits until element contains  css:#releaseNotes li:nth-of-type(1) time  ${date}
+    user waits until element contains  css:#releaseNotes li:nth-of-type(1) p     Test release note one
+
+Assert release owner can go to sign off page as contributor
+    [Tags]  HappyPath
+    user clicks link   Sign off
+    user waits until h2 is visible  Sign off
+    user waits until page contains button  Edit release status
+
+Check release owner can publish release
+    [Tags]  HappyPath
+    user clicks button  Edit release status
+    user waits until h2 is visible  Edit release status
+    user clicks radio   Approved for publication
+    user enters text into element  id:releaseStatusForm-internalReleaseNote  Approved by release owner
+    user clicks radio   As soon as possible
+    user clicks button   Update status
+
+Navigate to administration as bau1 to assign viewer only access
+    [Tags]  HappyPath    
+    user changes to bau1
+    user goes to url  %{ADMIN_URL}/administration/users
+
+Navigate to manage users as bau1
+    [Tags]  HappyPath
+    user clicks link  Manage  xpath://td[text()="ees-analyst1@education.gov.uk"]/..
+    user waits until page does not contain loading spinner
+    
+Remove release owner access from Analyst1 
+    [Tags]  HappyPath
+    user waits until element is enabled  css:[data-testid="remove-release-role-Approver"]
+    user clicks element  css:[data-testid="remove-release-role-Approver"]
+    user waits until page does not contain loading spinner
+
+Assign viewer only access to Analyst1
+    [Tags]  HappyPath
+    user waits until page does not contain loading spinner
```

Comment:
`user waits until page does not contain loading spinner` is called in the previous test case, so this can be removed